packages/core/deploy/036_deploy_across_ovm_bridge_deposit_box.js

```diff
@@ -0,0 +1,30 @@
+// This deploy script should be run on an optimism provider.
+const { getAddress } = require("@uma/contracts-node");
+const { ZERO_ADDRESS } = require("@uma/common");
+const func = async function (hre) {
```

Comment:
How does this get ignored for chains that don't have the right chainIds?

Reply:
In the form it has now, there is no ignoring; if you run the deploy script it will run no matter the chainID. previously, it had return early code. now, we just work around the problem with the default fixtures implementation.